app/models/purchase.rb

```diff
@@ -2,19 +2,24 @@
 #
 # Table name: purchases
 #
-#  id                    :bigint           not null, primary key
-#  amount_spent_in_cents :integer
-#  comment               :text
-#  issued_at             :datetime
-#  purchased_from        :string
-#  created_at            :datetime         not null
-#  updated_at            :datetime         not null
-#  organization_id       :integer
-#  storage_location_id   :integer
-#  vendor_id             :integer
+#  id                             :bigint           not null, primary key
+#  adult_incontinence_money_cents :integer          default(0), not null
+#  amount_spent_in_cents          :integer
+#  comment                        :text
+#  diapers_money_cents            :integer          default(0), not null
```

Comment:
This annotation looks out of sync with the latest changes. Could you re-run the migration and make these changes to match?

Reply:
Done!